.pylintrc

```diff
@@ -1,3 +1,17 @@
 [MASTER]
 ignore-paths=^.*/tests/.*$
-disable=unused-argument,too-few-public-methods,no-member,arguments-renamed,invalid-name
+disable=
+    arguments-renamed,
+    bad-continuation,
+    duplicate-code,
+    import-error,
+    invalid-name,
+    logging-format-interpolation,
+    logging-fstring-interpolation,
+    no-member,
+    no-self-use,
+    super-init-not-called,
+    too-few-public-methods,
+    unused-argument,
+    useless-super-delegation,
+    wrong-import-order,
```

Comment:
I notice there are some newcomers to this list, were those introduced by this PR?

Reply:
This is the sum total of the configs between screenpy and screenpy-selenium.  I figured if a setting was good for one project it was good for the other.

Reply:
I only put the ignores that i did in `screenpy` because they were necessary (and, thinking about it now, i think i might have been a little overzealous putting them as global ignores). I'd rather wait until it becomes obvious something _should_ be ignored for us to put it in here instead of aggregating them all, since they're based on the needs of other projects.

Reply:
Gotcha.  I will remove the configs added from screenpy.